content/advocates/advocates.json

```diff
@@ -3,13 +3,21 @@
     "name": "Rana Prathap Simh Mukthavaram",
     "image": "https://dl.airtable.com/.attachmentThumbnails/568134fea19c1e34a98c06cfd77a86ec/949489e7",
     "region": "Asia",
-    "location": "India"
+    "location": "India",
```

Comment:
We are not getting the location from the airtable, we are building it by joining the city and the country. It will be better to have mock data that fits the real one